pypsa/plot.py

```diff
@@ -32,8 +32,8 @@
 logger = logging.getLogger(__name__)
 
 
-__author__ = "Tom Brown (FIAS), Jonas Hoersch (FIAS)"
-__copyright__ = "Copyright 2015-2017 Tom Brown (FIAS), Jonas Hoersch (FIAS), GNU GPL 3"
+__author__ = "Tom Brown (FIAS), Jonas Hoersch (FIAS), Fabian Hofmann (FIAS), Fabian Neumann (KIT)"
+__copyright__ = "Copyright 2015-2019 Tom Brown (FIAS), Jonas Hoersch (FIAS); Copyright 2019 Fabian Hofmann (FIAS), Fabian Neumann (KIT), GNU GPL 3"
```

Comment:
```suggestion
__copyright__ = "Copyright 2015-2020 Tom Brown (FIAS), Jonas Hoersch (FIAS); Copyright 2019 Fabian Hofmann (FIAS), Fabian Neumann (KIT), GNU GPL 3"
```
;)

Reply:
We can also make this complete ;)
```suggestion
__copyright__ = "Copyright 2015-2020 Tom Brown (FIAS), Jonas Hoersch (FIAS); Copyright 2019-2020 Fabian Hofmann (FIAS), Fabian Neumann (KIT), GNU GPL 3"
```